src/AssetsPage/CopyButton/CopyButton.scss

```diff
@@ -0,0 +1,4 @@
+.button-width {
```

Comment:
I think this is about more than just the button width. Perhaps `copy-button` would be clearer?

Reply:
Sounds good to me!